composer.json

```diff
@@ -28,12 +28,13 @@
     "homepage": "https://docs.razorpay.com",
     "license": "MIT",
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.3.3",
```

Comment:
php 5.3.3 is a requirement for phpunit 4.8.0 
https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-4.8.0#requirements